README.md

```diff
@@ -46,3 +47,13 @@ Test your new function; I've provided some basic testing code in `code.test.js`.
 What is the worst-case asymptotic time complexity of your implementation? What
 is the worst-case asymptotic memory complexity? Add your answer, including your
 reasoning, to this markdown file.
+
+## My Response
+### Time Complexity
+My complexity should be $\Theta(n^22^n)$ because I have to iterate through one set of nested for loops ( $\Theta(n^2)$ ) during which 
```

Comment:
Still don't understand this -- the nested loop doesn't make the call to the function that does the memoization.

Reply:
I guess I'm not understanding the way I have to build this with the memoization, would it be (n * 2^n) instead? Since the HeldKarp call is happening inside the outer loop rather than the inside loop?

Reply:
It might help to think about the objects that are processed instead of trying to analyze the code line by line. In the worst case, your cache will be completely full, right? What does that mean in terms of the number of objects being stored in there? And what happens for each of those objects?